From: Hector Oron <zumbi@debian.org>
Date: Wed, 20 Feb 2019 13:25:36 +0100
Subject: load-versioned-libcc1

* d/p/load-versioned-libcc1.patch:
  - load libcc1.so.0 instead unversioned file.

* d/p/load-versioned-libcc1.patch:
  - load libcc1.so.0 instead unversioned file.
Author: Hector Oron <zumbi@debian.org>
---
 include/gcc-c-interface.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/gcc-c-interface.h b/include/gcc-c-interface.h
index d14472a..dae2f0a 100644
--- a/include/gcc-c-interface.h
+++ b/include/gcc-c-interface.h
@@ -188,7 +188,7 @@ struct gcc_c_context
 /* The name of the .so that the compiler builds.  We dlopen this
    later.  */
 
-#define GCC_C_FE_LIBCC libcc1.so
+#define GCC_C_FE_LIBCC libcc1.so.0
 
 /* The compiler exports a single initialization function.  This macro
    holds its name as a symbol.  */
